### PR TITLE
ci(Mergify): configuration update

### PR DESCRIPTION
**Pull request by suhaibmujahid**
This change has been made by @suhaibmujahid from Mergify config editor.

### CODE REVIEW

**Comment by marco-c** [comment]
How will this work when there are multiple dependency updates? Will it land all PRs together or in series?

**Comment by suhaibmujahid** [comment]
> How will this work when there are multiple dependency updates? Will it land all PRs together or in series?

My goal here is to try it out. 

My understanding that once a PR gets approved and passes the CI, it will be added to the merge queue. Next, the PRs will be merged in the order of entering the queue. However, we do not currently have a configuration to group the PRs in batches or to schedule the merge at a specific time, so the PRs should be merged as soon as passing all checks. 

In short, this just automates clicking on the merge button :)

**Review by marco-c — APPROVED** [review]
All right, let's see how it goes. I just want to make sure we don't go merging in series and so rebasing everything all the time :)

**Pull request merged**
